src/main/java/browserview/BrowserComponent.java

```diff
@@ -252,7 +253,7 @@ public void showIssue(String repoId, int id, boolean isPullRequest, boolean isFo
         runBrowserOperation(() -> scrollToBottom());
     }
 
-    public void jumpToComment(){
+    public void jumpToNewComment(){
```

Comment:
Maybe `jumpToCommentBox`? Also, rename the `JumpToCommentEvent` below and its handler.